### PR TITLE
begin with iavl v1.0.0

### PR DESCRIPTION
**Pull request by faddat**
Closes: #XXX

## What is the purpose of the change

Eliminate or reduce daily epoch lag

## Brief Changelog

Use iavl v1.0.0

## Testing and Verifying

All unit tests should pass when running make test in the root of cosmos-sdk folder.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? yes
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? not yet
  - How is the feature or change documented? it isn't yet

### CODE REVIEW

File: store/iavl/store.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func LoadStoreWithInitialVersion(db dbm.DB, logger log.Logger, key types.StoreKe
 	if err != nil {
 		return nil, err
 	}
-
+	istore.tree.(*immutableTree)
```
Comment (hieuvubk) [inline store/iavl/store.go]: Is there any purpose? If not we should remove.
cc @faddat

**Comment by julienrbrt** [comment]
IAVL v1 should be simpler to integrate that this in v0.47 btw: https://github.com/julienrbrt/cosmos-sdk-inj/compare/f/v0.47.3-inj-1...ac1b893f08942b97c308dc57716cf8c597ef967f

**Comment by julienrbrt** [comment]
IAVL v1 should be simpler to integrate in v0.47 btw: https://github.com/julienrbrt/cosmos-sdk-inj/compare/f/v0.47.3-inj-1...ac1b893f08942b97c308dc57716cf8c597ef967f

**Comment by faddat** [comment]
@julienrbrt thanks very much for the helpful comment!

**Pull request closed without merging**